Client/core/CGUI.cpp

```diff
@@ -748,7 +742,8 @@ void CLocalGUI::UpdateCursor()
             CCore::GetSingleton ().GetGame ()->GetPad ()->Clear ();*/
 
             // Restore the mouse cursor to its old position
-            SetCursorPos(m_StoredMousePosition.x, m_StoredMousePosition.y);
+            if (m_StoredMousePosition.x != -1 && m_StoredMousePosition.y != -1)
+                SetCursorPos(m_StoredMousePosition.x, m_StoredMousePosition.y);
```

Comment:
I wonder if this is what reverts the cursor to its original position when you toggle `showCursor` while ingame?

Reply:
Most likely.